### ISSUE TITLE
[BUG]: Doesn't work with aliases? 

### ISSUE THREAD

**Issue by oldjove** [JinMuGo/obsidian-go-up#1]
### **Describe the bug**

If the "up" link specified is an alias, using the "go up" command produces a "file cannot contain illegal characters" error, presumably because of the "|" in the alias

### **Expected behavior**

The command should open the file in question

### **To Reproduce**

Assign "up" property to an alias. Run "go up" command.

**Comment by JinmuGo** [comment]
thank you for bug submitting! i'll fix soon. 

give me more Additional context. 
1. using macOS or Windows?
2. Which Filename occur "file cannot contain illegal characters"?
- because i'm trying now with "JS" alias in "Javascript" Note
   it create new note name of "Javascript|JS" 

please answer the question? thank you

**Comment by oldjove** [comment]
Thanks!

1. I'm using Windows10
2. Seems any filename|alias will trigger this.

I tried your example: I created a file called "Javascript" with the alias "JS" (in the "aliases" property). I.e. "[[Javascript|JS]]"

In a second file, I added the "up" property as "[[Javascript|JS]]". When I run the "go up" command, I get the error about illegal characters in filenames. However, if "up" is set as "[[Javascript]]" only, the command works fine.

I don't quite understand why this error would occur because the command doesn't try to *create* a new file but just to trigger the link, but the error suggests that the command is trying to create a new file?? I'm no coder, so I don't understand the technicalities of this.

So yes, this is a minor issue given that it's always possible to use the actual filename and not the alias. I just don't see why it shouldn't work with an alias.

Does that help?
